tulip/mvp.py

```diff
@@ -0,0 +1,196 @@
+# Copyright (c) 2012-2015 by California Institute of Technology
```

Comment:
Year should be 2020.  Do you want to copyright as Caltech or UT Austin?

Reply:
Fixed